### ISSUE TITLE
Official documentation of InlineQueryResult discrepancy

### ISSUE THREAD

**Issue by Sarisan** [tdlib/telegram-bot-api#647]
Media size limitation passed with URL is different in each case.

`InlineQueryResultPhoto` - `5 MiB`
`InlineQueryResultMpeg4Gif` - `1 MiB`
`InlineQueryResultGif` - `1 MiB`
`InlineQueryResultDocument` - provides no size limitation

While in reality all support files up to `10 MiB` and up to `20 MiB` in local mode.
Is this a mistake or the documentation is outdated?

**Comment by levlam** [comment]
> While in reality all support files up to 10 MiB and up to 20 MiB in local mode.

This makes no sense. There is no a single difference in handling of files uploaded by URL in local mode compared to api.telegram.org.

**Comment by Sarisan** [comment]
> > While in reality all support files up to 10 MiB and up to 20 MiB in local mode.
> 
> This makes no sense. There is no a single difference in handling of files uploaded by URL in local mode compared to api.telegram.org.

Yes, specifically this makes no sense. It's always up to `20 MiB` despite the documentation.
